src/pages/beheer/aanvragen/index.tsx

```diff
@@ -42,16 +43,41 @@ const CPRequestsPage: NextPage = () => {
 				className="p-cp-requests"
 				pageTitle={t('pages/beheer/aanvragen/index___aanvragen')}
 			>
-				<ScrollableTabs
-					tabs={statusFilters}
-					onClick={(tabId) =>
-						setFilters({
-							...filters,
-							status: tabId.toString(),
-						})
-					}
-					variants={['rounded', 'light', 'bordered']}
-				/>
+				<div className="p-cp-requests__header">
+					<ScrollableTabs
+						className="p-cp-requests__status-filter"
+						tabs={statusFilters}
+						variants={['rounded', 'light', 'bordered']}
+						onClick={(tabId) =>
+							setFilters({
+								...filters,
+								status: tabId.toString(),
+							})
+						}
+					/>
+
+					<SearchBar
```

Comment:
The cursor is an arrow instead of a caret. This probably has to be fixed in the components library?
![image](https://user-images.githubusercontent.com/1710840/153185335-d81f43c7-fd40-4479-ba8c-69631e48370d.png)

![image](https://user-images.githubusercontent.com/1710840/153185440-1fc663b6-64a8-4ae4-abfb-73a0933801ae.png)

Reply:
Styling zit in de client dus kan hier gefixed worden.